src/main/java/frc/robot/subsystems/LED.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.constLED;
+import frc.robot.RobotMap.mapLED;
+
+import com.ctre.phoenix.led.Animation;
+import com.ctre.phoenix.led.CANdle;
+
+public class LED extends SubsystemBase {
+  CANdle LED;
+
+  /** Creates a new LED. */
+
+  public LED() {
+    LED = new CANdle(mapLED.LED_CAN);
+    configure();
+  }
+
+  public void configure() {
+    LED.configFactoryDefault();
+    LED.configBrightnessScalar(constLED.LED_BRIGHTNESS);
+  }
```

Comment:
We should be using the new CANdleConfiguration within constants and remove configure just like how we do motors